### ISSUE TITLE
Pomodoro timer in the same line are synced.

### ISSUE THREAD

**Issue by jguo-uw** [Roam-Research/issues#481]
**Describe the bug**

If applicable, add screenshots to help explain your problem. You can paste them in here and they will be uploaded automatically.

If you have a video of you experiencing the bug, that is also super helpful!

**To Reproduce**

4. See bug

**System Information:**
 - Roam version: [e.g 0.5.0-43911ff0, find it at the end of the help button pop up]
 - Device: [e.g. iPhone6, desktop computer]
 - OS: [e.g. iOS8.1, Windows, ]
 - Browser [e.g. Chrome, Safari, FireFox, Brave]

**Additional context**

**Comment by Bardia95** [comment]
Can you please elaborate?

**Comment by jguo-uw** [comment]
> Can you please elaborate?
I'm so sorry that I just realized my editing was not posted at all. I edited the main post already. Hope it works.

**Comment by Bardia95** [comment]
Ahh gotcha, we will look into this!

**Comment by Thomasb3** [comment]
Yes, this. I typically only use Pomodoro timers for longer tasks, so usually I'll need multiple timers before finishing. When I can finish in under 30 minutes, I'll just do it and not bother with timers. Since I can't add multiple functioning Pomodoro timers to a task, currently this feature has no practical use for me. I hope an option for adding multiple independent Pomodoro timers as jguo-uw described will be added.

**Comment by blueshrapnel** [comment]
I agree with @Thomasb3, it would be useful to have a string of timers, then when they're done you also have accurate feedback regarding the time and effort the task actually took.   As a work around I am adding the new timer on a new, indented block and then merging the blocks when they're done.
